wolox-books/src/components/NavBar.vue

```diff
@@ -0,0 +1,42 @@
+<template lang='pug'>
+  .header.row
+    .img-box.column
+      img.wolox-img(src='@/assets/LogoWolox.png')
+      span
+        | BOOKS
+    a.logout(href='javascript:void(0)' @click='logout')
```

Comment:
Use the anchor (tag `a`) for links. If this is going to be something you click on, then, use a button with `type='button'`.